packages/lab/stories/list-next/list.stories.tsx

```diff
@@ -1,32 +1,22 @@
 import { ComponentMeta, Story } from "@storybook/react";
-import { ListNext, ListItemNext, ListNextProps } from "../../src";
+import { ListItemNext, ListNext, ListNextProps } from "../../src";
 import { Button, FlexLayout } from "@salt-ds/core";
 import { useState } from "react";
+import { usStateExampleData } from "../assets/exampleData";
+import { ArrowDownIcon, ArrowUpIcon } from "@salt-ds/icons";
 
 export default {
```

Comment:
I don't think this is new in the PR but I'm noticing an issue in each of the stories. When you first mousedown on a ListItem (try it on an item which isn't the first) the first ListItem is highlighted, even if that isn't the one you are doing mousedown on.

Seems to only occur the first time an item is selected. Can fix in a separate PR if needed

Reply:
Ignore this. Hadn't fetched the latest